main.py

```diff
@@ -12,10 +13,15 @@
 
 def main():
     log.info(f"Python version: {sys.version}")
+    found = False
     while True:
         log.info("Loop")
         try:
             if win.find_game():
+                if not found:
+                    found = True
```

Comment:
Maybe move the found below to avoid the error after

Reply:
I suppose you can replace the `found = False` by `generate_temp_resolution()`